### ISSUE TITLE
Follow up of #36845.  "Quick Commands" text is not updated in Windows

### ISSUE THREAD

**Issue by MadhaviSeelam** [brave/brave-browser#41733]
### Description

Found while testing https://github.com/brave/brave-browser/issues/36845.  The text "Quick Commands" should be updated on all platforms including Windows. However, it's updated only on macOS platform

### Steps to reproduce

1. Install 1.72.83
2. launch Brave
3. press hamburger menu > more tools 

### Actual result

the text shows Quick commands

![Image](https://github.com/user-attachments/assets/d4996a43-79bb-439e-98f9-695836b796de)

### Expected result

Quick Commands text

![Image](https://github.com/user-attachments/assets/c1027502-1a73-4dc5-a53a-7f3cd5f01ce9)

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Brave | 1.72.83 Chromium: 130.0.6723.58 (Official Build) beta (64-bit)
-- | --
Revision | b49615851cd2dc4b1c3f5f13d3207d3c032bfd8d
OS | Windows 11 Version 23H2 (Build 22631.4317)

### Channel information

- [ ] release (stable)
- [x] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

@rebron @bsclifton 
cc: @brave/qa-team

**Comment by aadeetyeah** [comment]
Hi @MadhaviSeelam 
Newbie here would like to help/work on this.
As far as I understood you mean the "commands" first letter C should be capital in windows? Right?
As currently I am using following version of Brave on Windows :
[Version 1.73.91 Chromium: 131.0.6778.85 (Official Build) (64-bit)](https://brave.com/latest/)

And on this version as well I can see the same issue.

**Comment by bsclifton** [comment]
I don't think we need to fix this - but maybe it would be good to go over the whole menu and standardize as Windows has a mix of capital case and regular sentence casing
